ProjectSpace/explore/src/main/java/com/pwc/explore/RealTimeTest.java

```diff
@@ -0,0 +1,55 @@
+package com.pwc.explore;
+
+import android.content.Context;
+import android.util.Log;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+
+public class RealTimeTest {
+    private int numOfTry;
+    private int numOfCorrect;
+    private float accuracy;
+
+    public RealTimeTest() {
+        this.numOfTry = 0;
+        this.numOfCorrect = 0;
+        this.accuracy = 0.0f;
+    }
+
+    public int getNumOfTry() { return numOfTry; }
+    public void setNumOfTry(int numOfTry) { this.numOfTry = numOfTry; }
+    public int getNumOfCorrect() { return numOfCorrect; }
+    public void setNumOfCorrect(int numOfCorrect) { this.numOfCorrect = numOfCorrect; }
+    public float getAccuracy() { return accuracy; }
+    public void setAccuracy() { this.accuracy = (float) (this.numOfCorrect * 100) / this.numOfTry; }
+
+    public void saveResult(Context context) {
+        String TAG = "RealTimeTest";
+        try {
+            FileWriter out = new FileWriter(new File(context.getFilesDir(), "testResult.txt"));
```

Comment:
context.[getExternalFilesDir()](https://developer.android.com/training/data-storage/app-specific#external-access-files )

Reply:
Didn't notice that internal files can be only accessed by rooted devices and emulators. If saving data in an external directory, must users have an SD card?

Reply:
I believe it depends on the device. If I'm not wrong, generally devices allocate a [partition of its internal memory as external storage.](https://developer.android.com/training/data-storage/app-specific#external-select-location). 

This applies to the usual emulator virtual device we use, Pixel 3a. Furthermore, our _commsgaze_  contents are stored in this "external storage". 

We should be checking for space availability, which type of external storage to use etc, but I believe we can look into it later.

Reply:
Please, review the new commit I've done below.

Reply:
Look good!